Makefile

```diff
@@ -71,7 +71,7 @@ black: pipenv
 test: reports
 	export PYTHONPATH=$(PWD) && $(PIPENV) run pytest -v -s -x \
 		--junitxml=./reports/junit.xml \
-		tests/* | tee reports/pytest.log
+		tests/* || exit 1
```

Comment:
as we use the same tests locally, I may want to have these reports still. 
Should we use something like we use for mypy testing:

https://github.com/h2oai/h2o-llmstudio/blob/main/Makefile#L54-L55